runner.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+echo "Getting a fresh copy of the SDN list"
+
+curl -Lo sdn_advanced.xml https://www.treasury.gov/ofac/downloads/sanctions/1.0/sdn_advanced.xml
+
+echo "Generating CSV file of aggregated addresses"
+python3 generate-address-list.py -a -f CSV
+
+echo "Successfully generated CSV file of aggregated addresses"
+echo "Enjoy your day!"
```

Comment:
I guess we'd rather do this in the CI? 

https://github.com/0xB10C/ofac-sanctioned-digital-currency-addresses/blob/main/.github/workflows/generate-lists.yml

Reply:
it's an example I have shown not-IT user. 
if you want I can add as an example